addon/services/cookies.js

```diff
@@ -178,10 +178,6 @@ export default Service.extend({
         return acc;
       }
 
-      if (secure && !(protocol || '').match(/^https/)) {
```

Comment:
these changes seem unrelated to changing the CI system, no?

Reply:
The tests for this were failing on GH Actions and not on Travis and the check didn't really make sense anyway so I didn't bother investigating further. We can split the removal into a separate PR – let me know what your preference is there.

Reply:
> We can split the removal into a separate PR

I think that would be appropriate, yeah.

Reply:
see #447